app/helpers/operatingsystems_helper.rb

```diff
@@ -72,4 +72,12 @@ def os_habtm_family(type, obj)
     result = type.where(:os_family => obj.family)
     result.empty? ? type : result
   end
+
+  def os_default_templates_for_form(os)
+    if os.os_default_templates.any?(&:new_record?)
+      os.os_default_templates.sort { |o1, o2| o1.template_kind.name.downcase <=> o2.template_kind.name.downcase }
```

Comment:
How does this compare to using `sort_by`:
```suggestion
      os.os_default_templates.sort_by { |os| os.template_kind.name.downcase }
```

Reply:
yeah, updated